crates/taplo-cli/src/commands/queries.rs

```diff
@@ -93,52 +103,35 @@ impl<E: Environment> Taplo<E> {
                 }
             }
             crate::args::OutputFormat::Value => {
+                let mut buf = String::new();
+                let separator = cmd.separator.unwrap_or("\n".to_string());
```

Comment:
It doesn't look like this needs to be owned. What about borrowing only?

```suggestion
                let separator = cmd.separator.as_deref().unwrap_or("\n");
```